Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -89,6 +89,13 @@ configuration:
   name: hec_url
   required: false
   type: 0
+- defaultvalue: 3 days
```

Comment:
default value should be 10 minutes to avoid breaking bc

Reply:
done